tests/run/test_fstring.pyx

```diff
@@ -1169,9 +1168,10 @@ non-important content
         self.assertEqual(f'{0!=1}', 'True')
         self.assertEqual(f'{0<=1}', 'True')
         self.assertEqual(f'{0>=1}', 'False')
-        self.assertEqual(f'{(x:="5")}', '5')
+        # Walrus not implemented yet, skip
+        #self.assertEqual(f'{(x:="5")}', '5')
+        #self.assertEqual(f'{(x:=5)}', '5')
         self.assertEqual(x, '5')
-        self.assertEqual(f'{(x:=5)}', '5')
```

Comment:
Let's keep the order here. This file might get overwritten and manually merged from time to time with new copies from CPython's test suite. It's confusing if lines move around on our side.